app/src/main/java/com/vero/tasky/core/presentation/ErrorType.kt

```diff
@@ -0,0 +1,8 @@
+package com.vero.tasky.core.presentation
+
+import com.vero.tasky.R
+
+sealed class ErrorType(val message: Int) {
```

Comment:
Do you really need this tho?

Couldn't you just have a method in a util class (Object class) that's:

fun parsePasswordError(validation: PasswordValidationResult)

And return something based on that?